### DISCUSSION TITLE
Request for TTS App Expansion

### DISCUSSION THREAD

**Discussion by Mr-RyanMalik** [jing332/tts-server-android#110] [General]
Thank you for the excellent work on your TTS server app. I discovered it through Reddit, downloaded it from GitHub, and now use it daily to read blogs, articles, and other materials. It's a fantastic tool that has made my life easier.

However, I have a request: could you create a Chrome extension or desktop version of this app? Being able to read and listen to articles on my computer would be incredibly helpful. 

Once again, thank you so much for your hard work—I look forward to hearing back from you!

**Comment by DaveScream** [comment]
There is http api in side menu for this reasons. But i dont understanding how to use that

**Reply by Mr-RyanMalik** [reply]
I'm unsure how to use that API in the menu. Perhaps there's a Google Chrome extension that we could link it with, or a way to utilize it on my desktop computer. If anyone reading this knows how to do so, your help would be much appreciated. Additionally, if the developer of this fantastic app and system could assist us with configuration, we'd be grateful. Thank you!
